build/push-aws.sh

```diff
@@ -1,43 +1,59 @@
 #!/bin/bash
-# Push cockroach binaries to AWS.
+# Push binaries to AWS.
 # This is run by circle-ci after successful docker push.
 # Requisites:
+# - binaries must be statically linked by running build/build-static-binaries.sh
```

Comment:
maybe, although we should also check that they are linux binaries. That gets tedious